### PR TITLE
[WIP] Netlify Review

### PR DESCRIPTION
**Pull request by arichiv**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.5.1
+2.4.3
```
Comment (arichiv) [inline .ruby-version]: :-(

File: .travis.yml [hunk 2]
```diff
@@ -1,47 +1,9 @@
...
-    - git commit -m "ACTUALIZE ${TRAVIS_BUILD_NUMBER}"
-    - git push target HEAD:$TRAVIS_BRANCH
+cache: bundler
+script:
+  - ./scripts/test
+notifications:
+  email: false
+  slack:
+    rooms:
+      secure: EZJ/2mzTXTsNcH3TTKku0dzoZVroj7DRxsFvDTpYQwlVzu5F8DZlalUT7KXki+GvTMn5V81NsvdJ8fl9tutBiytOP/XGOrdKftbguBvRaTCwCm1mi1lwjOFdsJKVr+e6cQcFGZ1+Yef7aDuSpbjOquZRySifCJMiMB2r/tTi0yQKF9Xg/VsX5p6A4kqk9Vn/PDiH7gEJHrTpCpwWLZQLAsOnoE9DpczUgU9CcBz5zRxjCthw6RG36trb21aDjlSqU4JGBDl1rs5z4E2aIuUao0AUqI6C/DXDWdvKAYtMZNTX3yBAzG60vj3QbeclWNrOlBWg/9usfIaVWvC9OMjaaQILEIWNrVziqVpObN+eoFnPu1rni6cjPw1Fqf8MMlQCbUSeWua+BpS3wXBxCvn7xMw25RADpgIrMQSycut+MDu5iAmzyMRcQ0du0ttp7WZ/I0oO/bdM0llPEByG9WvmrlkCC5LaMA6m4dBpHouWG+TA9brkSVfK5v4fzhF+7T44jz9ByYyY7YYguPGNBq2ZeDln0iulIXoC8mHZQBWw9Q/kdL9C262+y6xgeTrdftJ8Mr/jNwpa1o5I72O/gZmAIL+kUilDClVzGX/E7PfZcWlRFhtpxsRZmgUr8fxJjHapEBXsWjqpC9EgHFlMSC/M9ck+QJHGw1uSwj60Le5U4qY=
```
Comment (arichiv) [inline .travis.yml]: this is often quite noisey

File: .ruby-version (hunk 1 above)
Comment (jgarber623-gov) [reply .ruby-version]: Not my first choice, but it's the newest [pre-installed Ruby](https://www.netlify.com/docs/#ruby) on Netlify's build platform:

> We also recommend specifying a version of Ruby that matches your build environment. You can set a Ruby version via a `/.ruby-version` file. You must choose from among [rvm](https://github.com/rvm/rvm)’s supported versions. As of this writing, the following versions are already installed and will create the fastest builds: 2.2.9 2.3.6 2.4.3

File: .travis.yml (hunk 2 above)
Comment (jgarber623-gov) [reply .travis.yml]: My recommendation is we create a `#code-mil-bots` channel for automated notifications and let folks opt in to the channel should they be interested in the information.

Comment (jordangov) [reply .travis.yml]: Agreed on the bot channel

**Pull request closed without merging**